sql/parse/parse.go

```diff
@@ -45,6 +45,8 @@ var (
 	ErrFrameEndUnboundedPreceding = errors.NewKind("frame end cannot be unbounded preceding")
 
 	ErrFrameStartUnboundedFollowing = errors.NewKind("frame start cannot be unbounded following")
+
+	ErrMultipleMixedDatabaseNotSupported = errors.NewKind("multiple mixed database %s tables not supported")
```

Comment:
Just use ErrUnsupportedFeature for this, and give it a message like "dropping tables on multiple databases in the same statement"